packages/devtools_app/test/screens/debugger/debugger_screen_test.dart

```diff
@@ -232,3 +252,20 @@ void main() {
     expect(state.line, testClassRef.location!.line);
   });
 }
+
+extension on CommonFinders {
```

Comment:
I think we prefer to have finder utils as just helper functions (e.g. https://github.com/flutter/devtools/blob/42391a90f67bbcf6791ceed9402caf399cf0cf98/packages/devtools_app/test/standalone_ui/ide_shared/property_editor_test.dart#L498C1-L502C1) instead of using an extension method

Reply:
Sure thing. Updated.